app/views/shared/check_answers/_linking_and_copying.html.erb

```diff
@@ -1,73 +1,55 @@
+              visually_hidden_text: [t(".section_linking.aria_prefix"), t(".section_linking.#{@legal_aid_application.lead_linked_application.link_type_code}")].join(" "),
+            )
           end
         end
-      end %>
-  <% if @legal_aid_application&.lead_linked_application&.confirm_link? && all_linked_applications_details(@legal_aid_application).present? %>
-    <div class="govuk-grid-row" id="app-check-your-answers__other_links">
-      <div class="govuk-grid-column-full">
-        <h2 class="govuk-heading-m"><%= t ".other_links_section.#{@legal_aid_application.lead_linked_application.link_type_code}" %></h2>
-      </div>
-    </div>
-    <%= govuk_summary_list(card: { title: t(".other_links_section.#{@legal_aid_application.lead_linked_application.link_type_code}") }, actions: false) do |summary_list|
+      end
+    end %>
+
+<% if @legal_aid_application&.lead_linked_application&.confirm_link? && all_linked_applications_details(@legal_aid_application).present? %>
+  <%= govuk_summary_card(title: t(".other_links_section.#{@legal_aid_application.lead_linked_application.link_type_code}")) do |card|
+        card.with_summary_list(actions: false) do |summary_list|
           all_linked_applications(@legal_aid_application).each do |application|
             summary_list.with_row do |row|
-              row.with_key(text: application.applicant.full_name, classes: "govuk-!-width-one-half")
-              row.with_value { sanitize(application.link_description) }
+              row.with_key(text: application.applicant.full_name, classes: "govuk-!-width-one-third")
+              row.with_value { sanitize(application.link_description_without_name) }
             end
           end
-        end %>
-  <% end %>
+        end
+      end %>
+<% end %>
 
-  <% if @legal_aid_application&.lead_linked_application&.confirm_link? && @legal_aid_application&.lead_linked_application&.link_type_code.eql?("FC_LEAD") %>
-    <div class="govuk-grid-row" id="app-check-your-answers__copying">
-      <div class="govuk-grid-column-two-thirds">
-        <h2 class="govuk-heading-m"><%= t ".section_copying.heading" %></h2>
-      </div>
-      <div class="govuk-grid-column-one-third">
-        <%= unless read_only
-              govuk_link_to(
-                t("generic.change"), providers_legal_aid_application_link_application_copy_path(@legal_aid_application),
-                class: "govuk-link-right"
-              )
-            end %>
-      </div>
-    </div>
-    <%= govuk_summary_list(card: { title: t(".section_copying.heading") },
-                           actions: !read_only,
-                           html_attributes: { id: "app-check-your-answers__copying_items" }) do |summary_list|
+<% if @legal_aid_application&.lead_linked_application&.confirm_link? && @legal_aid_application&.lead_linked_application&.link_type_code.eql?("FC_LEAD") %>
+  <%= govuk_summary_card(title: t(".section_copying.heading")) do |card|
+        card.with_action do
```

Comment:
This is a third way of handling actions on the same view, can they be normalised or are they all unique?

Reply:
Three I meant was 
```ruby
actions: !read_only
```
```ruby 
actions: false ```
and
```ruby 
unless read_only
  # add link
```
As long as they are all serving their different purposes, it fine.  I just wanted to make sure we weren't using different patterns randomly! :)